src/lib/api.js

```diff
@@ -60,7 +62,13 @@ const api = {
      * @return {Promise} - XHR promise
      */
     xhr(url, options = {}) {
-        return axios(url, api.filterOptions(options))
+        let transformResponse;
```

Comment:
Does having this `undefined` affect any other response types?

Reply:
should get filtered out on L70